### DISCUSSION TITLE
Numpy Kütüphanesi Nedir? Nasıl Kullanılır? | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#157] [Site]
# Numpy Kütüphanesi Nedir? Nasıl Kullanılır? | SonsuzUs Github

NumPy, Python programlama diline ait çok boyutlu dizilerle ve matrislerle çalışmamıza yardım eden ileri düzey matematiksel işlemler yapabileceğimiz bir kütüphanedir. Günümüzde özellikle veri bilimi üzerine çalışanlar başta olmak üzere Numpy, Python programlayanlar tarafından çok sık kullanılan bir kütüphanedir.

https://sonsuzus.github.io/posts/numpy-kutuphanesi-nedir-nasil-kullanilir/

**Comment by yigidi** [comment]
anladım
